app/modules/ws2812.c

```diff
@@ -399,19 +420,17 @@ static int ws2812_buffer_mix(lua_State* L) {
 
   size_t i;
   for (i = 0; i < cells; i++) {
-    int32_t val = 0;
+    int val = 0;
     for (src = 0; src < n_sources; src++) {
-      val += (int32_t)(source[src].values[i] * source[src].factor);
+      val += ((int)(source[src].values[i] * source[src].factor) >> 8);
     }
 
-    val >>= 8;
```

Comment:
I think that this was pulled out of the loop to enable smooth fading. Why move it back?

Reply:
Thanks. That was probably an accident, I did not intend to change this.